configs/components/rubygem-CFPropertyList.rb

```diff
@@ -10,8 +10,6 @@
   ### Maintained by update_gems automation ###
   pkg.version '4.0.0'
   pkg.sha256sum '385e7bbd7c27e176b23415275ca936ca0ece1727e5b890b08e36632076b00aff'
-  pkg.build_requires 'rubygem-base64'
```

Comment:
funfact: in newer rubies base64 isn't a std gem anymore. It's a shitshow .

Reply:
Yeah, we'll have to add a few back once we jump to 3.4/4.0.  It looks like they are trying to make more of the gems "bundled" versus "default", which makes updating them much easier at least.